ui/imports/shared/controls/chat/FetchMoreMessagesButton.qml

```diff
@@ -72,7 +72,8 @@ Item {
         horizontalAlignment: Text.AlignHCenter
         color: Style.current.secondaryText
         //% "before %1"
-        text: qsTrId("before--1").arg((nextMessageIndex > -1 ? new Date(nextMsgTimestamp * 1) : new Date()).toLocaleString(Qt.locale(localAppSettings.locale)))
+        text: qsTrId("before--1").arg(new Date(chatsModel.channelView.activeChannel.syncedTo * 1000).toLocaleString(Qt.locale(localAppSettings.locale)))
```

Comment:
```


string arg(value)

Returns a copy of this string with the lowest numbered place marker replaced by value, i.e., %1, %2, ..., %99. 
```
Sorry, but what `arg` this actually do? what expected?

Reply:
`qsTrId("before--1")` ends up being translated to `"before %1"`, then the `.arg(...)` will replace `%1` by a date